capa/v25.0.0/release.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: aws-25.0.0
```

Comment:
what is the naming convention of EKS cluster? will this be different? will it align and we verify somewhere else if it will be eks or capa?

Reply:
and what is the reason for not calling it capa-25.0.0?

Reply:
Not sure what is best, though I think the 2nd one is easier to read:

capa-v25.0.0
capa-v25.0.1
eks-v25.0.0
capz-v25.0.0
capz-v25.0.1
aks-v25.0.0

vs

aws-v25.0.0
aws-v25.0.1
eks-v25.0.0
azure-v25.0.0
azure-v25.0.1
aks-v25.0.0





[09:11](https://gigantic.slack.com/archives/D013546TA3D/p1718953873178509)

Reply:
IIUC long term `capa` will become `aws`, as soon as we completely migrate away from vintage https://github.com/giantswarm/roadmap/issues/1543.

Also long-term, even mid-term maybe, we should re-work Release CRD, so we can set provider name in some different way, like in spec, so CR name will not be that important like here, as here we are using Release CRD that has been designed for vintage, so we had to put the provider somewhere.